src/test/java/com/hocs/test/glue/CaseDecisionStepDefs.java

```diff
@@ -100,16 +99,16 @@ public void iCloseTheCaseWithADecisionOf(String status) {
         markUpDecision.getCaseId();
         switch (status.toUpperCase()) {
             case "REFER TO OGD":
-                markUpDecision.clickReferToOgdRadioButton();
+                page.clickOn(markUpDecision.referToOgdRadioButton);
                 break;
             case "NO REPLY NEEDED":
-                markUpDecision.clickNoReplyNeededRadioButton();
+                page.clickOn(markUpDecision.noReplyNeededRadioButton);
                 break;
             default:
                 pendingStep(status + " is not defined within " + getMethodName());
         }
-        markUpDecision.clickContinueButton();
-        markUpDecision.clickFinishButton();
+        page.clickOn(markUpDecision.continueButton);
+        page.clickOn(markUpDecision.finishButton);
 
```

Comment:
`page` should ideally display the name of the page where the button is being clicked.  As all page classes extend 'Page' it should always run but ofc ignore if you are avoiding stackOverflowErrors by doing it this way.